### PR TITLE
fix slice with None in shape

### PR DESCRIPTION
**Pull request by fumihwh**
If None in x shape, np.min will crash.

### CODE REVIEW

File: onnx_tf/backends/backend_v1.py [hunk 1]
```diff
@@ -789,8 +789,9 @@ def handle_slice(cls, node, input_dict):
 
     for i in range(slice_len):
       ends[i] = full_sizes[axes[i]] + ends[i] if ends[i] < 0 else ends[i]
-      ends[i] = np.min([full_sizes[axes[i]], ends[i]])
-      starts[i] = np.min([full_sizes[axes[i]], starts[i]])
+      if full_sizes[axes[i]] is not None:
```
Comment (tjingrant) [inline onnx_tf/backends/backend_v1.py]: Do we need to emit an error if None is encountered?

Comment (fumihwh) [reply onnx_tf/backends/backend_v1.py]: Even if None, we can also do slice with out of bounds risk.

Comment (fumihwh) [reply onnx_tf/backends/backend_v1.py]: @tjingrant Sure.

Comment (thatindiandude) [reply onnx_tf/backends/backend_v1.py]: Any updates?

Comment (tjingrant) [reply onnx_tf/backends/backend_v1.py]: I guess this seems to be the right thing to do for now to solve your problem. But let's discuss under your PR to see if there's anything high-level that needs to be done as well.

**Pull request merged**